ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -75,6 +75,9 @@ message GetTransactionsRequest {
   // Optional
   bool verbose = 5;
 
+  // Must be a valid domain ID
+  // Optional
+  string domain_id = 6;
```

Comment:
Maybe dumb question but if we have the domainID here, why having a separate update service? (rather than having this one returning transfers as well)

Reply:
field -> oneof breaks protobuf code generation compatibility (despite it should be backwards compatible)
so plainly for limiting migration impact for clients and ourselves.
Also see above: we might also leverage this for other changes. https://github.com/digital-asset/daml/pull/16760#discussion_r1176120611